Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfMagnificationFilterMode.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// Magnification filter mode.
+    /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/sampler.schema.json
+    /// </summary>
+    public enum GltfMagnificationFilterMode
+    {
+        None = 0,
```

Comment:
Can we get summary tags here?

Reply:
Synced offline: url should be enough, most enums don't have summary stubs from khronos gltf standard schema declaration.